program.c

```diff
@@ -11,6 +11,10 @@
 #ifdef JULIA_DEFINE_FAST_TLS // only available in Julia 0.7+
 JULIA_DEFINE_FAST_TLS()
 #endif
+// TODO(nhdaly): Would a better name be JL_IMAGE_NAME?
+#ifndef JULIA_LIB_NAME // Should be specified by -DJULIA_LIB_NAME="libname"
+#define JULIA_LIB_NAME "libhello"
+#endif
```

Comment:
Two questions:
1. Should I rename this variable to something more like JL_IMAGE_NAME? I'd like your input on this. :)

Reply:
Actually, thinking more, this should probably be something longer so it's even less likely to collide with anything someone might define in their ordinary c program for any reason... maybe `JULIAC_JL_IMAGE_NAME`?